keps/sig-scheduling/785-scheduler-component-config-api/README.md

```diff
@@ -296,5 +298,5 @@ N/A
 - 2020-05-08: KEP for beta graduation sent for review, including motivation,
   proposal, risks, test plan and graduation criteria.
 - 2020-05-13: KEP updated to remove v1alpha2 support.
-- 2021-07-08: Introducing `v1beta2`
-- 2021-08-06: Introducing `v1beta3` 
\ No newline at end of file
+- 2021-07-08: Introducing `v1beta2`.
+- 2021-08-06: Introducing `v1beta3`.
```

Comment:
add one more line:

- 2021-09-01: remove `v1beta1` and the legacy policy config API.

Reply:
correct me if not, i thought we only plan to remind users about the `v1beta1` deprecated policy, such as adding a warning log, or we want to remove the source code？

Reply:
this is just a log of updates to this kep

Reply:
I think we can safely remove `v1beta1` based on when it was introduced (1.19), but `v1beta2` has only existed as a replacement for 1 release, is that enough time for deprecation?

Reply:
@ahg-g I think the question is in the wording for the log... `2021-09-21: remove v1beta1...` vs `2021-09-21: mark v1beta1 deprecated`

Reply:
privide some messages, maybe not conformable. as [deprecation policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecation) says, `Beta feature to EOL: 3 months or 1 release (whichever is longer)`, we introduced `v1beta1` in release 1.19, more than 3 months or one release.

Reply:
I think we want to remove it completely. We can do so and it is likely better to reduce clutter and the overhead of maintaining three beta versions.

Reply:
updated both places, section under `introducing v1beta3` and `implementation history`.

Reply:
+1 to remove v1beta1 when v1beta3 gets introduced. It's not that typical/necessary to keep 3+ APIs at the same time.